package.json

```diff
@@ -202,8 +203,10 @@
         "node-fetch": "^2.6.1",
         "nodemon": "^2.0.7",
         "npm-run-all": "^4.1.5",
+        "perf_hooks": "^0.0.1",
```

Comment:
what is this for??

Reply:
But for reference, it's the node lib that allows you to use a version of `window.performance` on the server. It was an option I was trying out before using the method here.

Reply:
The recent press on substitution attacks make me a little wary of such package names
https://github.blog/2021-02-12-avoiding-npm-substitution-attacks/